scripts/deps-ppa/static_z3.sh

```diff
@@ -80,7 +80,9 @@ Vcs-Browser: https://github.com/Z3Prover/z3
 
 Package: libz3-static-dev
 Section: libdevel
-Architecture: any-i386 any-amd64
+Architecture: any-amd64
+Breaks: libz3-dev
+Replaces: libz3-dev
```

Comment:
@Marenz requested those - not sure it works like this without version, but we can try and see if it works.
Ref: https://www.debian.org/doc/debian-policy/ch-relationships.html#packages-which-break-other-packages-breaks - but I only quickly read through it to be honest :-).